### PR TITLE
Fix header color

### PR DESCRIPTION
**Pull request by 40n**
index

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -646,7 +646,7 @@
   }
   pre, h1, h2, h3, .header, table, table tr, table td, table th, blockquote,
   .pulse-graph, .btn:not(img), .minibutton, .social-count,
-  .discusion-topic-infobar, .box-header, .box-body, .timeline-comment-label,
+  .discusion-topic-infobar, .Box-header, .box-body, .timeline-comment-label,
```
Comment (silverwind) [inline github-dark.css]: You can remove this selector, it gets outruled by the rule you added below.

Comment (40n) [inline github-dark.css]: https://github.com/40n/GitHub-Dark/commit/1f907f2bb3259a2161dd91d39ab144a131304710

Comment (silverwind) [inline github-dark.css]: You can push to the same branch `fix`, changes will update in GitHub.

File: github-dark.css [hunk 2]
```diff
@@ -641,7 +641,7 @@
   .gist-quicksearch-result, .gist-quicksearch-result-group, .listgroup,
   .listgroup-item + .listgroup-item, .user-key-badge, .newsfeed-placeholder,
   .progress-bar-inline .progress-bar, .menu-heading, .border-gray-dark,
-  .discussion-item-review .blob-wrapper {
+  .discussion-item-review .blob-wrapper, .Box-header {
```
Comment (silverwind) [inline github-dark.css]: You can remove this rule. The rule below has the same specificity (1 class, 1 !important) so the last rule will always win.

Comment (40n) [inline github-dark.css]: Sorry, I was only half awake when I wrote this.

**Pull request merged**